content/posts/pandoc/pandoc.md

```diff
@@ -0,0 +1,334 @@
+---
+title: "How I escaped from Microsoft Office tools as a Computer Science student"
+date: 2019-10-09T00:35:20+07:00
+draft: true
+author: "kn"
+tags: ["pandoc", "presentation", "blog"]
+cover: "/posts/pandoc/images/the-powerpoint-was.jpg"
+keywords: ["pandoc"]
+description: "The Easiest Way to Make Presentations! (Pandoc + Markdown)"
+showFullContent: false
+---
+
+## Introduction and Motivation
+![](/posts/pandoc/images/mdVSw.png)
+
+I have used beamer quite a lot to prepare slides for both research and class presentations. I have started to use LaTeX to content in beamer presentation for two years. But for some time I've been concerned about the high ratio of markup to content in beamer presentations.  used to type thousands lines of code just to create a presentation slide which was tiring and time consuming. Thus, I did some research and found a combination of pandoc, markdown, and LaTeX to simplify my process. In my point of view, markdown in syntax is straightforward, easy to read and modify. Moreover, pandoc has the ability to convert markdown files into LaTeX, hence enabled me to customise my slides greatly later on.
+
+In this blog, I want to discuss about the creation of beamer presentations using a combination of *markdown, pandoc, and LaTeX*. This workflow offers the potential to reduce typing and increase readability of beamer presentation source code. After reading this blog, you might can create a slide by using both Markdown and LaTeX and without having to lie on Powerpoint. 
+
+## Why Pandoc For Presentation?
+
+**Presentation shouldn’t get in the way of content.**
+
+- With a word processor, you spend valuable time agonizing over
+what font size to make the section headings.
+
+    &rarr; With *Pandoc*, you just tell it to start a new section.
+
+- With a word processor, changing the formatting means you have to
+change each instance individually.
+
+    &rarr; With *Pandoc*, you just redefine the relevant commands
+
+- With a word processor, you have to carefully match any provided
+templates.
+
+    &rarr; With *Pandoc*, you can be sure you’ve fit the template, and switch
+templates easily.
+
+- Lower ratio of markup to content in beamer presentations comparing to LaTeX.
+- Beautiful.
+- Pretty Math Equation - From my point of view as a researcher.
+
+## Installation Guide
+
+**Integrated Software**
+
+Thus we want to make our slide looks like *LaTeX* typesets so a Basic LaTeX packages is required. \
+I strongly recommend not just installing the TeX compiler, but *all* TeX packages for your own convenience, so you have everything offline already. This will be a big download at first, but nowadays the storage space should be minimal.
```

Comment:
- so you have everything *available* offline ~~already~~
- ~~This will be a big download at first, but nowadays the storage space should be minimal~~